File: src/test/java/com/zy/StudentTest.java
package com.zy;

import com.Kotlin.Student;
import org.junit.Test;
import org.junit.jupiter.api.Assertions;

public class StudentTest {
    @Test
    public void highestScoreTest(){
        Student student = new Student("Tang",80,90);
        Assertions.assertEquals(90,student.highest());
    }
    @Test
    public void averageTest(){
        Student student = new Student("Tang",80,90);
        Assertions.assertEquals((80+90)/2,student.getAverage());
    }


}

File: src/test/kotlin/com/zy/StudentTest.kt
package com.zy

import com.Kotlin.Student
import org.junit.Test
import org.junit.jupiter.api.Assertions

class StudentTest(){
    @Test
    fun gradingTest(){
        val student = Student("Tang",60,90)
        Assertions.assertEquals('C',student.grading())
    } 

    @Test
    fun highestScoreTest(){
        val student = Student("Tang",60,90)
        Assertions.assertEquals(90,student.highest())
    }

    @Test
    fun averageTest(){
        val student = Student("Tang",60,90)
        Assertions.assertEquals((60+90)/2,student.getAverage())
    }


}